Example/Podfile

```diff
@@ -7,7 +7,7 @@ target 'Emission' do
                :subspecs => %w( Core RCTText RCTWebSocket RCTNetwork ),
                :inhibit_warnings => true
 
-  if ENV['ARTSY_STAFF_MEMBER'] != nil || ENV['CI'] != nil
+  if true #ENV['ARTSY_STAFF_MEMBER'] != nil || ENV['CI'] != nil
```

Comment:
Did this for now because with the CocoaPods app my env variables aren’t set 🤔

/cc @orta